docs/devctr-image.md

```diff
@@ -46,9 +46,31 @@ registry. The Firecracker CI suite must also be updated to use the new image.
 
 1. Build a new container image with the updated Dockerfile.
 
+   The below script is used to build a docker image.
+
    ```bash
-    docker build -t fcuvm -f tools/devctr/Dockerfile.x86_64 .
-    ```
+   tools/devtool build_devctr
+   ```
+
+   The script uses two files -
+
+    a) ```requirements.txt``` - This contains list of python packages
+    with their unbounded versions.
+
+    b) {arch-based}-requirements-freeze.txt
+    (eg ```x86-requirements-freeze.txt```) - This contains exhaustive
```

Comment:
I think we could phrase this a little clearer

What about:

`This file contains the exhaustive list of packages installed in the docker image, with their respective versions. Think of it like a Cargo.lock file for pip.`